modules/dnn/depth_prediction/c++/inference.cpp

```diff
@@ -0,0 +1,99 @@
+#include <iostream>

```

Comment:
What's the Openvino version do you use?
do you write a Makefile to make an executable application?

Reply:
我将把openvino的版本写入README中。
我还没有写Makefile，如果需要我可以学习写一个。
我将把openvino的版本写入README中。
我还没有写Makefile，如果需要我可以学习写一个。
I will write the version of openvino into readme.

I haven't written makefile yet. I can learn to write one if necessary.

Reply:
I haven't written makefile before, so it may take some time to write it.

So I first submitted a version to improve the openvino version information.

Reply:
please rename a meaningful patch name. and make sure the inference c code can be compiled.